net/pfSense-pkg-Avahi/files/etc/inc/priv/avahi.priv.inc

```diff
@@ -3,7 +3,7 @@
  * avahi.priv.inc
  *
  * part of pfSense (https://www.pfsense.org)
- * Copyright (c) 2015 Rubicon Communications, LLC (Netgate)
+ * Copyright (c) 2018 Denny Page
```

Comment:
Don't remove Rubicon Copyright

Reply:
I didn't remove the copyright--in effect it wasn't there to begin with. This package is a complete ground up writing, unrelated to the prior package. This particular file was initialized by copying and modifying the lldld.priv.inc.